src/util/fields.py

```diff
@@ -1008,6 +1008,34 @@ def adapt_dict(d):
             )
 
 
+# adapt depends for custom compute fields
+def adapt_depends(cr, model, old, new):
+    if not column_exists(cr, "ir_model_fields", "depends"):
+        # this field only appears in 9.0
+        return
+
+    target_model = model
+
+    match_old = r"\y{}\y".format(re.escape(old))
+    cr.execute(
+        """
+        SELECT id, model, depends
+          FROM ir_model_fields
+         WHERE state = 'manual'
+           AND depends ~ %s
+        """,
+        [match_old],
+    )
+    for id_, model, depends in cr.fetchall():
+        temp_depends = depends.split(",")
+        for i in range(len(temp_depends)):
+            domain = _adapt_one_domain(cr, target_model, old, new, model, [(temp_depends[i], "=", "depends")])
+            temp_depends[i] = domain[0][0] if domain else temp_depends[i]
```

Comment:
Useless write itself.
```suggestion
            if domain:
                temp_depends[i] = domain[0][0]
```